build.sbt

```diff
@@ -274,12 +228,16 @@ lazy val nonCrossedScalaVersionModules: Seq[ProjectReference] = Seq(
 lazy val coreModulesDeps: Seq[ClasspathDependency] =
   coreModules.map(ClasspathDependency(_, None))
 
+lazy val testModules: Seq[ProjectReference] = Seq(tests, `haskell-integration-tests`)
+
 lazy val root = project
   .in(file("."))
   .settings(name := "mu-scala")
   .settings(noPublishSettings)
   .aggregate(coreModules: _*)
   .dependsOn(coreModulesDeps: _*)
+  .aggregate(testModules: _*)
+  .dependsOn(testModules.map(ClasspathDependency(_, None)): _*)
```

Comment:
What does this line do? And why does the root project need to depend on anything? Isn't it just an aggregate of the child projects?

Reply:
It's for _compiling_ and _testing_ everything if you just do `sbt test`.

Reply:
Isn't that what `aggregate` does?

Reply:
Right https://github.com/higherkindness/mu-scala/pull/900/commits/2b04eb0d4a21fd4909308bb7e76cd0fdc72f50ad